### DISCUSSION TITLE
我是傻逼

### DISCUSSION THREAD

**Discussion by 123Windows31** [daizihan233/HanBot#26] [General]
<del>我是傻逼</del>

**Comment by daizihan233** [comment]
亲，标题不支持HTML，这里用的是MarkDown文档删除线你只要输入 \~~要加删除线的文字~~ 就可以了（）

**Comment by daizihan233** [comment]
标题只能是纯文字（）

**Comment by 123Windows31** [comment]
草，我还以为GitHub工作人员呢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "daizihan233/HanBot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年3月22日(星期二) 上午8:51
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [daizihan233/HanBot] <del&gt;fuck me</del&gt; (Discussion #26)





 
亲，标题不支持HTML，这里用的是MarkDown文档删除线你只要输入 ~~要加删除线的文字~~ 就可以了（）

**Comment by daizihan233** [comment]
草（）

**Comment by daizihan233** [comment]
标题就是纯文字，不支持HTML，也不支持MarkDown（）

**Comment by 123Windows31** [comment]
尼玛

**Reply by daizihan233** [reply]
尼玛
